EchoPro/computation/statistics.py

```diff
@@ -0,0 +1,128 @@
+import numpy as np
+import scipy.stats as st
+
+def stratified_transect_statistic( transects , 
+                                   strata , 
+                                   sample_fraction , 
+                                   replicates ):
+    
+    """
+    Calculates stratified mean statistics for a given transect
+    
+
+    Parameters
+    ----------
+    transects: pd.DataFrame
+        DataFrame comprising a variety of spatial metrics for transect data
+    strata: pd.DataFrame
+        DataFrame comprising summary features of latitude (INPFC) delimited strata
+    sample_fraction: np.float64
+        Value representing the proportion of ftransects that are resampled from the
+        overall dataset within each strata
+    replicates: int
+        The number of iterations/realizations used for bootstrapping
+
+    Notes
+    -----
+    This function calculates the stratified summary statistics for biomass within 
+    `EchoPro.survey.stratified_summary()`.
+    """ 
+    
+    ### Convert specific DataFrame columns to arrays for speed
+    distance = transects[ 'transect_distance' ].values
+    value = transects[ 'B_adult' ].values
```

Comment:
It seems that here the `B_adult` could be swapped out with other values too and the calculation is general. If that is true, how about adding an argument `param_name` and use `param_name="B_adult"` for the calculation?